LiveController_deployment.md

```diff
@@ -0,0 +1,91 @@
+## Machine prerequisites:
+====================================================
+liveController & liveRecorder can be installed in single or separate machines.
+
+## Software prerequisites:
+====================================================
+ - Centos 6.8+
+ - ffmpeg 3.0 -  (run build_scripts/build_ffmpeg.sh)
+
+
+ ## New installation of liveController version v1.16.1+:
```

Comment:
Why is 1.16 repeated here so many times? this is a new file, older branches won't even have it. If anything changes in newer branches, the doc will be edited, what's the point in specifying 1.16.x over and over?